x/evidence/src/genesis.rs

```diff
@@ -0,0 +1,49 @@
+use crate::{errors::GenesisStateError, Evidence};
+use gears::core::any::google::Any;
+use serde::{Deserialize, Serialize};
+
+/// GenesisState defines the evidence module's genesis state.
+#[derive(Debug, Clone, Default, PartialEq, Deserialize, Serialize)]
+pub struct GenesisState<E: Evidence>
+where
+    <E as std::convert::TryFrom<Any>>::Error: std::fmt::Debug,
+{
+    /// evidence defines all the evidence at genesis.
+    pub evidence: Evidences<E>,
+}
+
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize, Default)]
+#[serde(try_from = "Vec<Any>")]
+pub struct Evidences<E: Evidence>(Vec<E>)
```

Comment:
Would it make sense to have the `#[serde(try_from = "Vec<Any>")]` on the Gensis struct and avoid having the `Evidences` struct altogether?